File: mario-more/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int height;

    do
    {
        height = get_int("Height: ");
    }
    while(height > 8 || height < 1);

    for(int c = 0; c < height; c++)
    {
        for(int r = 0; r < c; r++)
        {
            printf("#");
        }
        printf("\n");
    }



}

File: sentimental-mario-more/mario.py
from cs50 import get_int

height = get_int("Height: ")

while height < 0 or height > 8:
    height = get_int("Height: ")

for i in range(height):
    for _ in range(height - i - 1):
        print(" ", end="")
    for _ in range(i + 1):
        print("#", end="")
    print("  ", end="")
    for _ in range(i + 1):
            print("#", end="")
    print()
